### ISSUE TITLE
Custom Resolution

### ISSUE THREAD

**Issue by adrardossa** [Sehyn/Wuwa_fpsunlocker#5]
is there any chance to add custom resolution and no border support?

**Comment by Sehyn** [comment]
I can add that of course, no border you mean fullscreen borderless? if so yes I can aswell

**Comment by ipisano** [comment]
More than a custom resolution I'm curious if it would be possible to increase the resolution scale, I play at 4K on a 27" display and I still notice aliasing... ATM I'm running DLDSR which means I have to change my desktop resolution (and DPI scaling because Windows changes those on its own) every time I start and finish playing, which annoyingly moves my open windows and even desktop icons around.
Actually, since I'm used with messing around with UE4/5 games configuration settings to tweak the TAA/TSR, the AO etc I'd like to get a version of this where we can actually make the modifications to the database on our own, if that's feasible.

**Comment by adrardossa** [comment]
yes, windowed borderless as well

**Comment by ipisano** [comment]
Wait, the game defaults to fullscreen (windowed) borderless by default unless you go out of your way to edit an ini file to get it to exclusive. Unless you want a window which is not fullscreen and doesn't have the Window top bar? I'm confused

Edit: something like this? (screenshot not mine)
![1ny2fm7bq5581](https://github.com/Sehyn/Wuwa_fpsunlocker/assets/18221957/fd9e55c4-8f7d-4d75-babc-f8149c481be2)

**Comment by adrardossa** [comment]
@ipisano yes thats correct, i would like to be able to do that especially without the top bar title. i was meant about the windowed borderless since the fullscreen already borderless

**Comment by adrardossa** [comment]
i was trying to look out for custom arguments UE4 but i wasnt able to found one.. the only one i know was unity its called -popup-window

**Comment by Sehyn** [comment]
You can do it using: https://framedsc.com/GeneralGuides/universal_ue4_consoleunlocker.htm

I can do it aswell but that means interacting with the game executable it may or may not cause a ban that's sensitive topic 

With this tool you have to attach to the game directly when you press start game

**Comment by adrardossa** [comment]
ah i see, i thought it was possible with the localstorage.db modifying as well

**Comment by Sehyn** [comment]
> ah i see, i thought it was possible with the localstorage.db modifying as well

You can try using this new version it allows you to modify anything you can play around:
https://github.com/Sehyn/Wuwa_fpsunlocker/releases/tag/V1.3

**Comment by Sehyn** [comment]
Ok I have found a way:
![image](https://github.com/Sehyn/Wuwa_fpsunlocker/assets/48888351/6598c77c-0b47-4991-9e34-3690b5f77e8e)

But that require me on interact with the executable and could cause bans or not let me know if I should implement it with disclaimer

**Comment by GSLinux** [comment]
I would say to include it with a disclaimer.

**Comment by adrardossa** [comment]
i agreed, better put a disclaimer which said u didnt take responsibility upon the user and so on

**Comment by adrardossa** [comment]
> > ah i see, i thought it was possible with the localstorage.db modifying as well
> 
> You can try using this new version it allows you to modify anything you can play around: https://github.com/Sehyn/Wuwa_fpsunlocker/releases/tag/V1.3

great, ill try later on and gave u the feedback
thanks!

**Comment by adrardossa** [comment]
apparently it doesnt work, it does the same as the one where if u edit on WindowsNoEditor folder

when entering the game it went back to normal settings
